src/Assets/TestPackages/dotnet-new/test_templates/TemplateWithConditionalParameters/Test.cs

```diff
@@ -2,10 +2,10 @@
 // value of paramA: placeholderA
 // value of paramB: placeholderB
 
-//#if( A_enabled )
+//#if( paramA )
 // A is enabled
 //#endif
```

Comment:
nit

```suggestion
//#if( paramA )
// A is enabled
//#elseif
// A is disabled
//#endif
```

just for clarity